.changes/next-release/bugfix-codeartifactlogin-92237.json

```diff
@@ -0,0 +1,5 @@
+{
+  "type": "bugfix",
+  "category": "``codeartifact login``",
+  "description": "\"Fix parsing of dotnet output for aws codeartifact login command; fixes `#6197 <https://github.com/aws/aws-cli/issues/6197>`__\""
```

Comment:
It looks like there are duplicate double quotes in the description. We should remove these.